mriqc/workflows/functional.py

```diff
@@ -55,18 +55,19 @@ def fmri_qc_workflow(dataset, settings, name='funcMRIQC'):
     # Workflow --------------------------------------------------------
 
     # 1. HMC: head motion correct
-    hmcwf = hmc_mcflirt()
-    if settings.get('hmc_afni', False):
+    if settings.get('hmc_fsl', False):
```

Comment:
What's the advantage? Isn't this conditional logic more clear? The added bonus is that we never construct the afni workflow if mcflirt was requested.